### DISCUSSION TITLE
CQWW Experience with QLog

### DISCUSSION THREAD

**Discussion by oh2gba** [foldynl/QLog#489] [General]
Hello! Just wanted to share my experience with last weekend's CQWW SSB contest. I used QLog, and it worked great. It handled a few pile-ups smoothly. While it may lack some features compared to other contest logging software, it was perfect for a casual contester like me. I understand that contesting isn’t the main focus of QLog, but it’s actually quite useful for it. With a few more tweaks, it could even be a serious contender—especially since there isn’t really an alternative on Linux at the moment. Exporting logs was easy too; just filter by date and use an online tool to convert the ADIF log file to Cabrillo. Thumbs up for QLog and the occasional contest!

**Comment by foldynl** [comment]
I also participated in the CQWW SSB and  I got a lot of improvement notes for QLog ;-) I can only say one thing about it... just wait for the next release. There is still a lot of work to do, but hopefully, the changes will be in a positive direction.
